m01_introduction/01_gallery.rst

```diff
@@ -0,0 +1,158 @@
+.. The roles  should be created at the beginning of the document. Global roles can be created in the `global.rst` file
+
+.. include:: /global.rst
+
+.. Chapter content
+
+Gallery of features
+===================
+
+.. styled-topic::
+
+  Main questions:
+    What are the main features of A+?
+
+  Topics:
+    In this section we will talk about:
+
+    * `Front page`_
+    * `Exercises`_
+    * `Services`_
+
+  Material:
+    In this chapter you won't need extra material.
+
+  Requirements:
+    There are no requirements for this chapter.
+
+  Estimated working time:
+    20min.
+
+
+
+This section is a visual index of exercises and services that have been used
+with A+.
+
+.. _front-page:
```

Comment:
If I remember correctly, it is possible to make links to section headings without adding a custom label before the heading. Though, it is OK to add custom labels too and you can then use different wording in the label than in the heading.